src/Auth.js

```diff
@@ -1,13 +1,15 @@
 const ACCESS_TOKEN_KEY = 'access_token';
 const EXPIRES_AT_KEY = 'expires_at';
 const USER_ID = 'user_id';
+const BEARER_TOKEN = 'cfd-members-auth0IdToken';
```

Comment:
There was already a line in index.js that takes the bearer token in localStorage and uses it in the Authorization header. However, I couldn't find a place where the bearer token was actually set, so I put it here.